linsolve/linsolve.py

```diff
@@ -446,7 +451,8 @@ def __init__(self, data, wgts={}, sparse=False, **kwargs):
                 Variables with trailing underscores '_' are interpreted as complex conjugates (e.g. x*y_ 
                 parses as x * y.conj()).
             wgts: Dictionary that maps equation strings from data to real weights to apply to each 
-                equation. Any equation not in wgts gets a weight of 1.0. Defaults to {} (i.e. all 1.0s).
+                equation. Weights are treated as 1/sigma^2. Any equation not in wgts gets a weight of 1.0. 
```

Comment:
Seems to me if you provide wgts, it should be an error not to have an equation assigned a weight.

Reply:
Turns out that's actually what the code was doing and the documentation was just out of date. I'll fix it.

Reply:
(see the verify_weights function)